pom.xml

```diff
@@ -7,11 +7,40 @@
     <groupId>org.example</groupId>
     <artifactId>Homework28</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
 
     <properties>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <java.version>17</java.version>
     </properties>
+    <dependencies>
+
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.13.2</version>
```

Comment:
JUnit 4? Seriously? Use JUnit Jupiter instead, check the testing lesson video if you need to refresh your memory on this topic.